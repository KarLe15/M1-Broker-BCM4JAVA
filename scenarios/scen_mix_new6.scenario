## operation, component, scenario, uri
CREATE,subscriber,scen_mix_new6,subscriber_1
CREATE,subscriber,scen_mix_new6,subscriber_2
CREATE,publisher,scen_mix_new6,publisher_1
CREATE,publisher,scen_mix_new6,publisher_2
