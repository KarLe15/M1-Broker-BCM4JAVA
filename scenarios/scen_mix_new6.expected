{
"subscriber_1" : 332,
"subscriber_2" : 332
}
